features/ownership/request.feature

```diff
@@ -13,6 +13,7 @@ Feature: Test requesting ownership
     And the JSON response at "itemId" should be "%{organizerId}"
     And the JSON response at "itemType" should be "organizer"
     And the JSON response at "ownerId" should be "auth0|64089494e980aedd96740212"
+    And the JSON response at "ownerEmail" should be "dev+e2etest@publiq.be"
     And the JSON response at "requesterId" should be "7a583ed3-cbc1-481d-93b1-d80fff0174dd"
```

Comment:
Isn't it mainly the requester's email that is needed?

simon@publiq.be asked ownership of org x -> approve / reject

Reply:
No, the requester is internal information. For example, admin `jonas@publiq.be` requested ownership for user `simon@publiq.be`.

Reply:
Ok 👍🏻 @vhande for clarification